File: test/fib.c

#include <stdio.h>

int fib(int n) {
    if (n < 2) {
        return n;
    } else {
        return fib(n-1) + fib(n-2);
    }
}

int main() {
    putn(fib(40));
    putchar('\n');
    return 0;
}

File: test/fib.js

const more = (n) => {
    return fib(n-1) + fib(n-2);
};

const fib = (n) => {
    if (n < 2) {
        return n;
    } else {
        return more(n);
    }
};

console.log(fib(40));
